src/app/routes/home/getInitialData/index.js

```diff
@@ -16,14 +17,38 @@ const transformJson = pipe(
   squashTopStories,
   filterGroupsWithoutStraplines,
 );
+
 const getRadioScheduleToggle = path(['frontPageRadioSchedule', 'enabled']);
 const getRadioSchedulePosition = path(['frontPageRadioSchedule', 'value']);
 
+const fetchElectionsOembed = async service => {
+  const usElectionOembedPath = `/${service}/election/us2020/results/oembed`;
+
+  try {
+    const { json, status } = await fetchPageData({
+      path: usElectionOembedPath,
+    });
+
+    if (json && status === 200) {
```

Comment:
Could we use a static variable? https://github.com/bbc/simorgh/blob/latest/src/app/lib/statusCodes.const.js#L1

Reply:
done